package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cloudnative/health",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
no need to do this, I'll change the version when I cut a release

Reply:
Reverted